core/arch/arm/mm/tee_pager.c

```diff
@@ -1066,6 +1069,8 @@ static struct tee_pager_pmem *tee_pager_get_page(struct tee_pager_area *area)
 		area_set_entry(pmem->area, pmem->pgidx, 0, 0);
 		pgt_dec_used_entries(pmem->area->pgt);
 		tlbi_mva_allasid(area_idx2va(pmem->area, pmem->pgidx));
+		if (a & (TEE_MATTR_PX | TEE_MATTR_UX))
+			cache_op_inner(ICACHE_INVALIDATE, NULL, 0);
```

Comment:
Ok, thanks. Maybe this deserves the same kind of comment (or one will someday likely propose an "optim" here). 

However, since aliased pages are not executable, I wonder if the dual virtual addresses (target and aliased) of the page processed here is an issue.